integration_test/test_videoroom/test/integration/containerised_test.exs

```diff
@@ -0,0 +1,203 @@
+defmodule TestVideoroom.Integration.ContainerisedTest do
+  use TestVideoroomWeb.ConnCase, async: false
+
+  require Logger
+
+  @moduletag :containerised
+
+  # in milliseconds
+  @max_test_duration 400_000
+
+  # Name of shared volume folder
+  @shared_folder "shared"
+
+  # Necessary for peer ID mapping
+  @sender_stats_tag :local_stream_stats
+
+  @packet_loss_browser_id "browser0"
+  @packet_loss_tags_to_check [:after_warmup, :after_applying_packet_loss_on_one_user]
+  # How many result entries will be discarded when checking the ratio between frames
+  # received from regular browsers and frames received from the one with packet loss applied
+  @packet_loss_results_ratio_warmup 5
+  # Max allowed difference between received frames from separate browsers, at each stat entry
+  @max_frame_count_difference 10
+
+  @tag timeout: @max_test_duration
+  @tag :packet_loss_test
+  test "packet loss on one browser" do
+    TestVideoroom.Integration.ResultReceiver.start_link(browser_count: 3, parent: self())
+
```

Comment:
I still think it would be nicer to have one Elixir node outside of the containers connected to them via distributed Erlang. You could move the logic responsible for testing outside of the `mediaserver` docker, get rid of the `run_packet_loss_test.sh` script and writing to file to enable packet loss hack and do everything from Elixir, also logs from test wouldn't be mixed with docker containers logs. It seems like it was decided to take care of this in later PR, but idk if this is the later PR.

Reply:
I completely agree, but that was outside the scope of this PR, and will have to be done in the future ;)

Reply:
That should be the next thing after this PR